tasks/libs/common/check_tools_version.py

```diff
@@ -74,9 +91,14 @@ def check_tools_version(ctx: Context, tools_list: list[str]) -> bool:
         else:
             current_v, expected_v = tools_versions[tool]['current_v'], tools_versions[tool]['expected_v']
             if current_v != expected_v:
-                is_expected_versions = False
                 print(
-                    f"Warning: Expecting {tool} '{expected_v}' but you have {tool} '{current_v}'. Please fix this as you might encounter issues using the tooling.",
+                    color_message(
+                        f"Expecting {tool} '{expected_v}' but you have {tool} '{current_v}'. Please fix this as you might encounter issues using the tooling.",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should you suggest also gere to run `inv -e setup`?

Reply:
Good point, fixed with [d9f11cb](https://github.com/DataDog/datadog-agent/pull/29802/commits/d9f11cb15a85c4171e55cb6d6d4c6a33e01c9799).